app/views/users/show.html.erb

```diff
@@ -14,32 +14,35 @@
     <% if @user.token == nil %>
       <p>No github stuff</p>
     <% else %>
-    <h1><%= @repos.count %> Recent Repos</h1>
+      <h1><%= @repos.count %> Recent Repos</h1>
       <% @repos.each do |repo| %>
         <ul>
           <li><a href= "<%= repo.url %>"><%= repo.name %></li></a>
         </ul>
       <% end %>
-      <section class="github_followers">
-        <h1><%= @followers.length %> Followers</h1>
-        <% @followers.each do |follower| %>
+    <% end %>
+
```

Comment:
I was getting syntax errors so I spaced things out for better readability (spelling?)